diff -urN FreeMat-4.1-Source/libs/libMath/libDynBlas/blas_dyn_link.cpp FreeMat-4.1-Source-patched/libs/libMath/libDynBlas/blas_dyn_link.cpp
--- FreeMat-4.1-Source/libs/libMath/libDynBlas/blas_dyn_link.cpp	2011-11-27 00:27:43.228793406 +0000
+++ FreeMat-4.1-Source-patched/libs/libMath/libDynBlas/blas_dyn_link.cpp	2012-12-01 02:53:25.054224687 +0000
@@ -124,7 +124,7 @@
 
 void BlasWrapper::DiscoverBlasLibrary( void )
 {
-	QSettings *libsettings = new QSettings( QCoreApplication::applicationDirPath()+"/blas.ini", QSettings::IniFormat );
+	QSettings *libsettings = new QSettings( "@@FREEMAT_DIR@@/blas.ini", QSettings::IniFormat );
 	libsettings->beginGroup( ComputerType().c_str() );
 
 	QStringList libs = libsettings->childGroups();
diff -urN FreeMat-4.1-Source/src/CMakeLists.txt FreeMat-4.1-Source-patched/src/CMakeLists.txt
--- FreeMat-4.1-Source/src/CMakeLists.txt	2012-12-01 02:57:06.543612051 +0000
+++ FreeMat-4.1-Source-patched/src/CMakeLists.txt	2012-12-01 02:57:11.436120633 +0000
@@ -183,7 +183,7 @@
 if (APPLE)
 INSTALL( FILES ${PROJECT_SOURCE_DIR}/tools/blas.ini DESTINATION FreeMat.app/Contents/MacOS)
 else()
-INSTALL( FILES ${PROJECT_SOURCE_DIR}/tools/blas.ini DESTINATION bin )
+INSTALL( FILES ${PROJECT_SOURCE_DIR}/tools/blas.ini DESTINATION share/FreeMat )
 endif()
 
 
